apps/web/public/static/locales/en/common.json

```diff
@@ -1534,7 +1534,7 @@
   "url_taken": "This URL is already taken",
   "problem_registering_domain": "There was a problem with registering the subdomain, please try again or contact an administrator",
   "team_publish": "Publish team",
-  "number_text_notifications": "Phone number (Text notifications)",
+  "number_text_notifications": "Phone number (SMS notifications)",
```

Comment:
That doesn't fix it. We need a new translation key `number_sms_notifications`
We want to have both

Reply:
Okay.I will update it

Reply:
I have updated the PR with new changes.